File: FullCart-WebAPI/FullCart-WebAPI/Models/Item.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace FullCart_WebAPI.Models
{
    public class Item
    {
        [Key]
        public int Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public string ImageUrl {  get; set; } = string.Empty;
        public int Price {  get; set; }
        public int Quantity { get; set; }
        public int BrandId {  get; set; }

        [ForeignKey("BrandId")]
        public Brand? Brand { get; set; }

        public int CategoryId { get; set; }

        [ForeignKey("CategoryId")]
        public Category? Category { get; set; }
    }
}

File: FullCart-UI/shared/models/Item.ts
import { Brand } from "./Brand";
import { Category } from "./Category";

export interface Item {
  id: number;
  name: string;
  description: string;
  imageUrl: string;
  price: number;
  quantity: number;
  brandId: number;
  categoryId: number;
  brand?: Brand;
  category?: Category;
}
